Sources/XcodeProj/Scheme/XCScheme+LaunchAction.swift

```diff
@@ -43,9 +43,15 @@ public extension XCScheme {
         public var launchStyle: Style
         public var askForAppToLaunch: Bool?
         public var pathRunnable: PathRunnable? {
-          // For backwards compatibility
-          runnable as? PathRunnable
+            // For backwards compatibility
+            get {
+                runnable as? PathRunnable
+            }
+            set {
+                self.pathRunnable = newValue
```

Comment:
@pepicrft Looks like this was supposed to be `self.runnable = newValue`. Xcode is warning about infinite recursion since it is essentially calling it's own setter.